docs/data-access/data-download.md

```diff
@@ -0,0 +1,89 @@
+---
+id: data-download
+title: How to Download Data
+---
+
+The ARGO Data Platform uses the score-cient as a file download manager. Score facilitates the transfer of data with resumable download and built in BAM/CRAM slicing to make data download transfer fast and smooth.
+
+Please note:
+
+- Downloads are done in part and can be paused/resumed as needed
+- The score-client will automatically resume downloads if interrupted or paused briefly
+
+## Searching for Files
+
+Use the Platform [File Repository](https://platform.icgc-argo.org/repository) to search for a file set of interest. Query results can be narrowed down by selecting specific file values in the facet filters on the left side of the page. Once you have a file set identified, click `Download > File Manifest` on the top right of the table to download a TSV file manifest.
+
+The file manifest will contain a listing of the files that matched your query, along with some additional metatdata to assist in file identification. This file manifest will be used by the score-client as a list of files to download.
+
+> NOTE: Clinical data can be downloaded by any user. In order to download controlled molecular data, you **must have ICGC DACO approval**. Learn more about the [DACO application process here](data-access), or [apply for DACO approval here](https://icgc.org/daco).
+
+## Installing the Score-Client
+
+The score-client can be run in different ways depending on your operating system or setup:
+
+- If you are on Windows, use the Docker distribution
+- If you are on a Unix system (IOS/Linux) you can use the Docker distribution, or score-client directly.
```

Comment:
is it clear what "Docker distribution" is to readers?

Reply:
yes i think so.  - added `score-client` before just to be more clear.

Reply:
I mean, I would have no idea what a "docker" is...do researchers?

Reply:
its not really a question we get in the current icgc usage.  its a technology that the people working with this data are familiar with.